.travis.yml

```diff
@@ -0,0 +1,25 @@
+---
+sudo: required
+dist: trusty
```

Comment:
Travis uses 12.04 by default and 14.04 as beta. There's only a few choices re base distros to run on. More advanced distro testing is probably best done with Docker.